scripts/test_hdf5_index_11364.py

```diff
@@ -0,0 +1,37 @@
+import pandas as pd
+import os
+
+def create_test_file():
+    col_nums = [0]
+    df = pd.DataFrame({"V1":["a","b","c","d","e", "aaaah!!!"], 
```

Comment:
you need to craft a test along the lines of the tests in ``pandas/io/tests/test_pytables.py``